From f425735af6b034467dd23590d38e0ca15a074dec Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Thu, 6 Mar 2025 09:28:01 +0100
Subject: [PATCH] netavark: lock version of cargo iptables to 0.5.2

Recently the v0.5.3 was released and compilation kept failing on it.
Additionally even in repo of netavark the version 0.5.2 is specified
in lock file.
---
 .../0001-Fix-version-of-iptables.patch        | 25 +++++++++++++++++++
 1 file changed, 25 insertions(+)
 create mode 100644 net/netavark/patches/0001-Fix-version-of-iptables.patch

diff --git a/net/netavark/patches/0001-Fix-version-of-iptables.patch b/net/netavark/patches/0001-Fix-version-of-iptables.patch
new file mode 100644
index 000000000..185d54a2a
--- /dev/null
+++ b/net/netavark/patches/0001-Fix-version-of-iptables.patch
@@ -0,0 +1,25 @@
+From c02ec79ec48ae23b8d12f807cfd65f1db96f2052 Mon Sep 17 00:00:00 2001
+From: Richard Muzik <richard.muzik@nic.cz>
+Date: Thu, 6 Mar 2025 09:07:04 +0100
+Subject: [PATCH] Fix version of iptables
+
+---
+ Cargo.toml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Cargo.toml b/Cargo.toml
+index ea8701d..c31ba72 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -34,7 +34,7 @@ anyhow = "1.0.86"
+ clap = { version = "~4.4.12", features = ["derive", "env"] }
+ env_logger = "0.11.5"
+ ipnet = { version = "2.9.0", features = ["serde"] }
+-iptables = "0.5.2"
++iptables = "<=0.5.2"
+ libc = "0.2.154"
+ log = "0.4.22"
+ serde = { version = "1.0.199", features = ["derive"], optional = true }
+-- 
+2.48.1
+
-- 
2.48.1

